tests/server/api/conftest.py

```diff
@@ -407,3 +407,138 @@ async def dataset_with_revisions(session):
     )
     session.add(example_7_revision_9)
     await session.flush()
+
+
+@pytest.fixture
+async def dataset_with_experiments_without_runs(session, empty_dataset):
+    experiment_0 = models.Experiment(
+        id=0,
```

Comment:
Can we omit the explicit IDs? They prevent different fixtures from composing together.

Reply:
can you explain? I think it's important to make the id explicit so we know how to reference it since when we load the fixture the id would otherwise be dynamically generated. fixtures are already composable unless we conflict on and load them at the same time

Reply:
I mean so that we can use more than one fixture at once without having conflicts on the hard-coded IDs.

Reply:
This is non-blocking.

Reply:
if we need two fixtures to interact I think they should have different ids, setting it in advance is for explicitness so tests can either select the correct (or incorrect) id as desired

Reply:
Okay.